.dev-lib

```diff
@@ -6,6 +6,7 @@ PROJECT_SLUG=amp
 README_MD_TITLE="AMP Plugin for WordPress"
 DEV_LIB_SKIP="$DEV_LIB_SKIP,jshint"
 CHECK_SCOPE=all
+PLUGIN_FILE="$(pwd)/amp.php"
```

Comment:
To make this a bit more robust, it would probably be best if `wp-dev-lib` supplied the directory name and the environment variable here was only set as `amp.php`. Just thinking that as it stands right now, if the user is not currently in the plugin's root directory, then this could fail to work as expected. Granted this is unlikely. Nevertheless, it seems unnecessary to always require adding `$(pwd)/` every time.